mobile/lib/screens/home_page.dart

```diff
@@ -219,23 +204,13 @@ class _HomePageState extends State<HomePage> {
   void _onItemTapped(int index) {
     switch (index) {
       case 0:
-        context.read<NearbyLocationBloc>().add(const CheckNearbyLocations());
+        context
+            .read<NearbyLocationBloc>()
+            .add(const SearchLocationAirQuality());
```

Comment:
Searching for the air quality of current location when a user switches to the home page